README.md

```diff
@@ -91,6 +100,18 @@ Run `nix-tree` on your current nixos system:
 nix-tree /nix/var/nix/profiles/system
 ```
 
+Query the binary cache before download, with the `--store` option:
+
+```bash
+# Query the runtime dependency of `stellarium` (2 GiB closure) without download
+nix eval --raw 'nixpkgs#stellarium' | xargs -o nix-tree --store https://cache.nixos.org
```

Comment:
This didn't work for me - I got the error: 

```
nix-tree: user error (Invalid path: /nix/store/va01q4ln9y41q1rn126pm1slq78v1m0w-stellarium-23.4. Make sure that it is built, or pass '--derivation' if you want to work on the derivation.)
mosscap:nix-tree utdemir$ nix eval --raw 'nixpkgs#stellarium' | xargs -o cabal run nix-tree -- --store https://cache.nixos.org --derivation
error: '/nix/store/va01q4ln9y41q1rn126pm1slq78v1m0w-stellarium-23.4' does not have a known deriver
nix-tree: Received ExitFailure 1 when running
Raw command: nix path-info --json --derivation --store https://cache.nixos.org/ --extra-experimental-features "nix-command flakes" /nix/store/va01q4ln9y41q1rn126pm1slq78v1m0w-stellarium-23.4
```

Reply:
Maybe your system already had `stellarium` downloaded?

Reply:
Oh no, that's weird... I haven't installed stellarium; also, I tried other random packages by monkey typing on my keyboard :laughing: and they all work... I am not sure what happened...

Maybe it's related to my environment?
```console
$ nix run nixpkgs#nix-info -- -m
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.8-2-MANJARO, Manjaro Linux, noversion, rolling`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - nixpkgs: `/home/bryan/.nix-defexpr/channels/nixpkgs`
```

Reply:
What happens if `nix eval --raw 'nixpkgs#stellarium.outPath'` with the `.outPath` ?

Reply:
That command does output okay, same output with/without `.outPath`. But when I pass that output path to `nix-tree` I get the error:

```
mosscap:nix-tree utdemir$ nix eval --raw 'nixpkgs#stellarium'; echo
/nix/store/va01q4ln9y41q1rn126pm1slq78v1m0w-stellarium-23.4
mosscap:nix-tree utdemir$ nix eval --raw 'nixpkgs#stellarium.outPath'; echo
/nix/store/va01q4ln9y41q1rn126pm1slq78v1m0w-stellarium-23.4
mosscap:nix-tree utdemir$ cabal run nix-tree -- --store https://cache.nixos.org /nix/store/va01q4ln9y41q1rn126pm1slq78v1m0w-stellarium-23.4 --derivation
error: '/nix/store/va01q4ln9y41q1rn126pm1slq78v1m0w-stellarium-23.4' does not have a known deriver
nix-tree: Received ExitFailure 1 when running
Raw command: nix path-info --json --derivation --store https://cache.nixos.org --extra-experimental-features "nix-command flakes" /nix/store/va01q4ln9y41q1rn126pm1slq78v1m0w-stellarium-23.4

mosscap:nix-tree utdemir$ cabal run nix-tree -- --store https://cache.nixos.org /nix/store/va01q4ln9y41q1rn126pm1slq78v1m0w-stellarium-23.4 
don't know how to build these paths:
  /nix/store/va01q4ln9y41q1rn126pm1slq78v1m0w-stellarium-23.4
nix-tree: user error (Invalid path: /nix/store/va01q4ln9y41q1rn126pm1slq78v1m0w-stellarium-23.4. Make sure that it is built, or pass '--derivation' if you want to work on the derivation.)
```

I guess this might be because I am on a mac and that derivation does not build for it hence the cache is not populated. I'll try with a package that works on a mac.

Reply:
Okay that was it! It works! (I tried with `emacs` assuming people got it working)

Reply:
Oh, nice! Awesome!